### PR TITLE
widget: Implement immediate capability negotiation

### PR DESCRIPTION
**Pull request by jplatte**
The "regular" capability negotiation where the driver first waits for the widget to send content-loaded is not yet implemented, because this way we only have to implement `toWidget` message handling, `fromWidget` is stubbed out.

### CODE REVIEW

**Review by daniel-abramov — APPROVED** [review]

**Comment by jplatte** [comment]
Merging without waiting for the extra review. Has one approval, coverage improved and we want to build on this which is a bit easier when it's merged.

A post-merge review would still be appreciated though.

**Pull request merged**
